bflibrary/src/x86-win-sdl2/sscreen.c

```diff
@@ -480,6 +480,10 @@ TbResult LbScreenSetupAnyMode(TbScreenMode mode, TbScreenCoord width,
         if (new_fullscreen_flags == 0)
         {
             SDL_SetWindowSize(lbWindow, mdWidth, mdHeight);
+            if ((SDL_getenv("SDL_VIDEO_CENTERED")) && (strcmp((SDL_getenv("SDL_VIDEO_CENTERED")), "1") == 0))
+            {
+                SDL_SetWindowPosition(lbWindow, SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED);
+            }
         }
     }
 
```

Comment:
We also need to change `SDL_WINDOWPOS_UNDEFINED` below.. Maybe instead of storing the bool value, we should store:
```
int windowpos;

windowpos = SDL_WINDOWPOS_UNDEFINED;
if (SDL_GetHintBoolean("SDL_VIDEO_CENTERED", SDL_FALSE))
    windowpos = SDL_WINDOWPOS_CENTERED;

if (windowpos != SDL_WINDOWPOS_UNDEFINED)
SDL_SetWindowPosition(lbWindow, windowpos, windowpos);

    lbWindow = SDL_CreateWindow(lbDrawAreaTitle, windowpos, windowpos,
          mdWidth, mdHeight, sdlFlags);
```